lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -86,6 +86,10 @@ def __init__(self, time_idx, coordinate_idx, image):
         self.coordinate_idx = list(coordinate_idx)
         self._image = image
 
+    @classmethod
+    def from_kymolinedata(cls, kymolinedata, image):
+        return cls(kymolinedata.time_idx, kymolinedata.coordinate_idx, image)
+
```

Comment:
I think this is good, it makes a clear connection between the two classes. Should it maybe be a private method? I don't think we want users using this directly since `KymoLineData` is meant only for internal use, but `KymoLine` is exposed as results from the kymotracker.

Reply:
Private now.